Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
I disagree with this change because it's not an official image.
This base image comes from :
- Docker Hub : https://hub.docker.com/r/isuper/java-oracle/tags/
- Github : https://github.com/orgisuper/docker